spec/factories/persons.rb

```diff
@@ -13,8 +13,7 @@
     springshare_id { "0123-4567-8901" }
     # Add related objects in create.
     # e.g.
-    #   let(:building) { FactoryBot.create(:building) }
-    #   let(:space) { FactoryBot.create(:space, building: building) }
+    spaces { [FactoryBot.create(:space, building: FactoryBot.create(:building))] }
```

Comment:
@skng5 Check this out. Was able to set up the person factory to handle creating it's own dependencies by inlining the calls to factorybot. Ugly, but less ugly than having them in every singe place where we use the person factory.

Reply:
Nice! Thank you.